### DISCUSSION TITLE
Installation instructions result in error

### DISCUSSION THREAD

**Discussion by quixot1c** [PixArt-alpha/PixArt-alpha#62] [Q&A] [answered]
The installation instructions as presented in the README.md result in an error. Specifically trying to run `pip install torch==2.1.1 torchvision==0.16.1 torchaudio==2.1.1 --index-url https://download.pytorch.org/whl/cu117` results in
```
ERROR: Could not find a version that satisfies the requirement torch==2.1.1 (from versions: 1.13.0+cu117, 1.13.1+cu117, 2.0.0+cu117, 2.0.1+cu117)
ERROR: No matching distribution found for torch==2.1.1
```
It seems that a newer version of CUDA is required for the 2.1.1 package versions?

**Comment by lawrence-cj** [comment, marked as answer]
It will work fine on torch-2.0.1. Fell free to change. Thx for noticing, already fixed it.
